.github/workflows/lint.yml

```
@@ -1,20 +1,32 @@
+            cache: "pip"
+
+        - name: "Install requirements"
+          run: python3 -m pip install -r requirements.txt
+
+        - name: "Lint"
+          run: python3 -m ruff check .
 
-      - name: ⬛ Run Black
-        uses: psf/black@stable
+        - name: "Format"
+          run: python3 -m ruff format . --check
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)